com.unity.cinemachine/Samples~/Shared Assets/UI/SamplesDynamicUI.cs

```diff
@@ -0,0 +1,90 @@
+using System;
```

Comment:
CameraMagnets and BrainUpdateModes UI is broken.  Buttons don't respond and and overlap with debug UI

![image](https://user-images.githubusercontent.com/6424658/224779196-e4fa9cd6-cccc-4f59-974d-b8a53301c154.png)

Reply:
Yes, the scenes are not updated, so the references of the buttons got lost. I'll do that once we are ok with everything, so I don't have scene conflicts with main.

The debug UI overlap can be fixed easily once we have runtime uitoolkit for that too. My plan is to have this approved, then I open another PR targeting this, which will add runtime uitoolkit for debug text. What do you think?

Reply:
yes, that's fine

Reply:
This PR https://github.com/Unity-Technologies/com.unity.cinemachine/pull/831 is going to address the overlap issue.